### PR TITLE
[GESSO-150] Fixed bugs outlined in ticket. Re: 3rd bug, opted for jus…

### PR DESCRIPTION
**Pull request by juranosaurustechs**
…t changing bg color rather than refactoring.

### CODE REVIEW

File: _starter-kit/_patterns/03-components/pager/_pager.scss [hunk 1]
```diff
@@ -50,18 +50,14 @@ $pager-bg-color-active: #d6d7d9;
...
 .pager__link {
   @extend %pager__link;
   display: inline-block;
   text-decoration: none;
 
   &:focus,
-  &:hover {
+  &:hover,
+  .pager__item--current & {
     background-color: $pager-bg-color-active;
```
Comment (dcmouyard) [inline _starter-kit/_patterns/03-components/pager/_pager.scss]: Why did you move the `.pager__item--current` selector here?

**Review by dcmouyard — CHANGES_REQUESTED** [review]
The `pager.twig` file also needs to be updated to remove the `<a>` tag on current items.

The markup for pagers doesn’t match what we had in the D7 version:

```html
<nav role="navigation" aria-labelledby="pagination-heading">
  <h2 id="pagination-heading" class="visually-hidden">Pagination</h2>
  <ul class="pager js-pager__items">
    <li class="pager__item pager__item--first">
      <a class="pager__link pager__link--first" href="#" title="Go to first page">
        <span class="visually-hidden">First page</span>
        <span aria-hidden="true">|«</span>
      </a>
    </li>
    <li class="pager__item pager__item--previous">
      <a class="pager__link pager__link--previous" href="#" title="Go to previous page" rel="prev">
        <span class="visually-hidden">Previous page</span>
        <span aria-hidden="true">‹</span>
      </a>
    </li>
    <li class="pager__item pager__item--ellipsis" role="presentation">…</li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        2
      </a>
    </li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        3
      </a>
    </li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        4
      </a>
    </li>
    <li class="pager__item"><a class="pager__link" href="#">
      <span class="visually-hidden">Page&nbsp;</span>
      5
    </a>
    </li>
    <li class="pager__item pager__item--current">
      <span class="visually-hidden">Currently on page&nbsp;</span>
      6
    </li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        7
      </a>
    </li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        8
      </a>
    </li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        9
      </a>
    </li>
    <li class="pager__item">
      <a class="pager__link" href="#">
        <span class="visually-hidden">Page&nbsp;</span>
        10
      </a>
    </li>
    <li class="pager__item pager__item--ellipsis" role="presentation">…</li>
    <li class="pager__item pager__item--next">
      <a class="pager__link pager__link--next" href="#" title="Go to next page" rel="next">
        <span class="visually-hidden">Next page</span>
        <span aria-hidden="true">›</span>
      </a>
    </li>
    <li class="pager__item pager__item--last">
      <a class="pager__link pager__link--last" href="#" title="Go to last page">
        <span class="visually-hidden">Last page</span>
        <span aria-hidden="true">»|</span>
      </a>
    </li>
  </ul>
</nav>
```

**Comment by juranosaurustechs** [comment]
@dcmouyard I didn't change the current item to be a link. I was just trying to get the current class to match BEM and have it look right. Is is a problem having the current pager item a link?
